src/web_ui/frontend/.env

```diff
@@ -0,0 +1 @@
+FAST_REFRESH = false
```

Comment:
Let's pull the port number assignment out of package.json and readme.md files.

```
    "start": "PORT=4060 react-scripts start",
    "windows-start": "set PORT=4060 && react-scripts start",
```

Reply:
Good idea.
Commit: 825d5bc006694263e19081ad0c22c7492604892b